sys/net/gnrc/pktbuf_static/gnrc_pktbuf_static.c

```diff
@@ -437,6 +437,7 @@ static void *_pktbuf_alloc(size_t size)
         DEBUG("pktbuf: no space left in packet buffer\n");
         return NULL;
     }
+    /* _unused_t struct would fit => add new space at ptr */
     if (sizeof(_unused_t) > (ptr->size - size)) {
```

Comment:
shouldn't this be `if ((_unused_t != NULL) && (sizeof(_unused_t) > (ptr->size - size)))`, compare L454 where `_first_unused = NULL;`?